platform-ajax/src/main/java/com/softicar/platform/ajax/engine/AjaxDomEngine.java

```diff
@@ -536,6 +542,11 @@ private void JS_removeNodeAttribute(IDomNode node, String name) {
 		updateCodeJS.appendStatement(JS_getNode(node) + ".removeAttribute('" + name + "');");
 	}
 
+	private void JS_setAttribute(String attribute, String value) {
```

Comment:
hmm, isn't this method redundant? shouldn't be the first occasion where we set an attribute

Reply:
I did not find any such method, please give an example :)

Reply:
ok